app/Services/ApiServices/InventoryService.php

```
@@ -2,69 +2,116 @@
     protected $apiUrl;
 
     protected $authToken;
 
-    public function __construct($authToken)
+    public function __construct()
     {
-        $this->authToken = $authToken;
         $this->apiUrl = config('services.url.inventory_api');
+        $this->authToken = config('services.sigma.secret_key');
+        if (empty($this->authToken)) {
+            throw new \InvalidArgumentException('SECRET KEY is not configured');
```

PHPMD (2.15.0)

Missing class import via use statement (line '20', column '23'). (Clean Code Rules)

(MissingImport)